llama-index-core/llama_index/core/command_line/rag.py

```diff
@@ -7,15 +7,14 @@
 from typing import Any, Callable, Dict, Optional, Union, cast
 
 from llama_index.core import (
-    Response,
-    ServiceContext,
```

Comment:
Both of these imports work fine for me, but sure, more specific imports is fine too

Reply:
I would suggest making a fresh venv if those imports were causing issues tbh